_includes/helpers.js

```diff
@@ -0,0 +1,141 @@
+function buildTable(insideNode, confs, includeCFP) {
+	var table = document.createElement("table");
+	insideNode.appendChild(table);
+	var thead = document.createElement("thead");
+	table.appendChild(thead);
+	var headRow = document.createElement("tr");
+	thead.appendChild(headRow);
+	
+	var nameHead = document.createElement("th");
+	nameHead.appendChild(document.createTextNode("Name"));
+	headRow.appendChild(nameHead);
+	var dateHead = document.createElement("th");
+	dateHead.appendChild(document.createTextNode("Date"));
+	headRow.appendChild(dateHead);
+	var placeHead = document.createElement("th");
+	placeHead.appendChild(document.createTextNode("Place"));
+	headRow.appendChild(placeHead);
+
+	if (includeCFP === true) {
+		var cfpHead = document.createElement("th");
+		cfpHead.appendChild(document.createTextNode("CFP Deadline"));
+		headRow.appendChild(cfpHead);
+	}
+	
+	var tbody = document.createElement("tbody");
+	table.appendChild(tbody);
+	
+	for (var i = 0; i < confs.length; i++) {
+		tbody.appendChild(buildRow(confs[i], includeCFP));
+	}
+
+}
+
+var months = ["", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+
+function buildRow(conference, includeCFP) {
+	var tr = document.createElement("tr");
+	
+	var nameCell = document.createElement("td");
+	var conferenceName = document.createTextNode(conference.name);
+	if (conference.link !== null) {
+		var linkNode = document.createElement("a");
+		linkNode.setAttribute("href", conference.link);
+		linkNode.setAttribute("title", conference.name);
+		linkNode.appendChild(conferenceName);
+		nameCell.appendChild(linkNode);
+	} else {
+		nameCell.appendChild(conferenceName);
+	}
+	tr.appendChild(nameCell);
+	
+	var dateNode = document.createElement("td");
+	var start = conference.start;
+	var end = conference.end;
+	var dateString = "";
+	if (start.year === end.year && start.month === end.month && start.day === end.day) {
+		// 1-day conference
+		dateString = months[start.month] + " " + start.day + ", " + start.year;
+	} else  if (start.month !== end.month) {
+		// conference spans the end of a month
+		var startString = months[start.month] + " " + start.day;
```

Comment:
What do you think of using string interpolation (e.g. template literals) for some of these? 

```suggestion
		const startString = `${months[start.month]} ${start.day}`;
````

Reply:
For a total JS noobie like me, this is a bit more confusing about what's going on. IMO, using `+` is a bit more approachable for anyone who's looking to contribute.

Reply:
Alright, I'm cool with it :) The only downside is that using string concatenation is a bit more error-prone, but It's not a huge deal